src/wovnio/wovnphp/Headers.php

```diff
@@ -320,6 +327,23 @@ public function requestOut()
                 $this->env['SERVER_NAME'] = $this->removeLang($this->env['SERVER_NAME']);
                 break;
 
+            case 'custom_domain':
```

Comment:
Just something nice to have: in requestOut() there are a lot of duplicate code fragments, maybe we can extract them out like so:

    private function uriRemoveLang()
    {
        if (isset($this->env['REQUEST_URI'])) {
            $this->env['REQUEST_URI'] = $this->removeLang($this->env['REQUEST_URI']);
        }
        if (isset($this->env['REDIRECT_URL'])) {
            $this->env['REDIRECT_URL'] = $this->removeLang($this->env['REDIRECT_URL']);
        }
        if (isset($this->env['HTTP_X_FORWARDED_REQUEST_URI'])) {
            $this->env['HTTP_X_FORWARDED_REQUEST_URI'] = $this->removeLang($this->env['HTTP_X_FORWARDED_REQUEST_URI']);
        }
    }

    private function hostRemoveLang()
    {
        if ($this->store->settings['use_proxy'] && isset($this->env['HTTP_X_FORWARDED_HOST'])) {
            $this->env['HTTP_X_FORWARDED_HOST'] = $this->removeLang($this->env['HTTP_X_FORWARDED_HOST']);
        }
        $this->env['HTTP_HOST'] = $this->removeLang($this->env['HTTP_HOST']);
        $this->env['SERVER_NAME'] = $this->removeLang($this->env['SERVER_NAME']);
    }

Reply:
agree, I was kind of expecting we would have `removeLangFromPath`, `removeLangFromHost` functions